EpiAware/docs/src/showcase/replications/chatzilena-2019/index.jl

```diff
@@ -0,0 +1,622 @@
+using EpiAware
+
+# ╔═╡ a4710701-6315-459d-b677-f24b77ff3e80
+using Turing
+
+# ╔═╡ 7263d714-2ce4-4d57-8881-6b60db018dd5
+using OrdinaryDiffEq, SciMLSensitivity #ODE solvers and adjoint methods
+
+# ╔═╡ 261420cd-4650-402b-b126-7a431f93f37e
+using Distributions, Statistics #Statistics packages
+
+# ╔═╡ 9c19a98b-a08b-4560-966d-61ff0ece2ad5
+using CSV, DataFramesMeta #Data wrangling
+
+# ╔═╡ 3897e773-ed07-4860-bb62-35605d0dacb0
+using CairoMakie, PairPlots
+
+# ╔═╡ 14641441-dbea-4fdf-88e0-64a57da60ef7
+using ReverseDiff #Automatic differentiation backend
+
+# ╔═╡ a0d91258-8ab5-4adc-98f2-8f17b4bd685c
+begin #Date utility and set Random seed
+    using Dates
+    using Random
+    Random.seed!(1234)
+end
+
+# ╔═╡ 33384fc6-7cca-11ef-3567-ab7df9200cde
+md"
+# Example: Statistical inference for ODE-based infectious disease models
+# Introduction
+## What are we going to do in this Vignette
+In this vignette, we'll demonstrate how to use `EpiAware` in conjunction with [SciML ecosystem](https://sciml.ai/) for Bayesian inference of infectious disease dynamics. The model and data is heavily based on [Contemporary statistical inference for infectious disease models using Stan _Chatzilena et al. 2019_](https://www.sciencedirect.com/science/article/pii/S1755436519300325).
+
+We'll cover the following key points:
+
+1. Defining the deterministic ODE model from Chatzilena et al section 2.2.2 using SciML ODE functionality and an `EpiAware` observation model.
+2. Build on this to define the stochastic ODE model from Chatzilena et al section 2.2.3 using an `EpiAware` observation model.
+3. Fitting the deterministic ODE model to data from an Influenza outbreak in an English boarding school.
+4. Fitting the stochastic ODE model to data from an Influenza outbreak in an English boarding school.
+
+## What might I need to know before starting
+
+This vignette builds on concepts from `EpiAware` observation models and a familarity with the `SciML` and `Turing` ecosystems would be useful but not essential.
+
+## Packages used in this vignette
+
+Alongside the `EpiAware` package we will use the `OrdinaryDiffEq` and `SciMLSensitivity` packages for interfacing with `SciML` ecosystem; this is a lower dependency usage of `DifferentialEquations.jl` that, respectively, exposes ODE solvers and adjoint methods for ODE solvees; that is the method of propagating parameter derivatives through functions containing ODE solutions. Bayesian inference will be done with `NUTS` from the `Turing` ecosystem. We will also use the `CairoMakie` package for plotting and `DataFramesMeta` for data manipulation.
+"
+
+# ╔═╡ 943b82ec-b4dc-4537-8183-d6c73cd74a37
+md"
+# Single population SIR model
+
+As mentioned in _Chatzilena et al_ disease spread is frequently modelled in terms
+of ODE-based models. The study population is divided into compartments representing a specific stage of the epidemic status. In this case, susceptible, infected, and recovered individuals.
```

Comment:
is this line break intentionnal?

Reply:
ha fair. One option we could take going forward is to put all new sentences on a new line. Makes it easier to see changes in git and maybe plays nicer with precommit. Downside is very boring.

Reply:
For a person with a taut readable style this would be no problem...

Reply:
Ok I've done this except for bullets